src/extension/default_extension_ek.cairo

```diff
@@ -202,8 +203,16 @@ mod DefaultExtensionEK {
     }
 
     #[constructor]
-    fn constructor(ref self: ContractState, singleton: ContractAddress, v_token_class_hash: felt252) {
+    fn constructor(
+        ref self: ContractState,
+        singleton: ContractAddress,
+        core: ContractAddress,
```

Comment:
I moved core from a constant to the constructor so that it makes writing the tests easier.